packages/components/src/disabled/test/index.tsx

```diff
@@ -47,18 +18,14 @@ describe( 'Disabled', () => {
 	);
 
 	it( 'will disable all fields', () => {
-		render(
+		const { container } = render(
 			<Disabled>
 				<Form />
 			</Disabled>
 		);
 
-		const input = screen.getByRole( 'textbox' );
-		const contentEditable = screen.getByTitle( 'edit my content' );
-		expect( input ).toBeDisabled();
-		expect( contentEditable ).toHaveAttribute( 'contenteditable', 'false' );
-		expect( contentEditable ).not.toHaveAttribute( 'tabindex' );
-		expect( contentEditable ).not.toHaveAttribute( 'disabled' );
```

Comment:
For me these were testing internal implementation anyway and not good tests so that's why I removed them. That said, I agree that "trying to interact" is a better test here.

Reply:
Agreed! The previous tests were definitely testing implementation details too